app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/return_user.scala.html

```diff
@@ -37,7 +37,7 @@ <h2 class="heading-medium">@messages("cds.checking-status-page.heading2")</h2>
         <li>@messages("cds.checking-status-page.bullet3")</li>
     </ul>
 
-    <a class="button" href='@uk.gov.hmrc.eoricommoncomponent.frontend.controllers.routes.EmailController.form(Service.NullService, journey)' role="button">@messages("cds.navigation.continue")</a>
```

Comment:
This was one of the "fixes" I was making for "re-enable register" journey - I think it should be Service.CDS or your new equivalent.

Reply:
Hmm, but this redirect is to the subscription journey.

Why here we don't need to pass a current service but CDS?

Reply:
Sorry.  My turn to comment on the wrong file!

Reply:
You might be right.  Let me look at my "register" branch

Reply:
return_user screen is connected to ReturnUserController which is connected to subscribe endpoint.

But in general the issue can be with endpoint. And endpoint need to be changed to register